contracts/interfaces/ILensHub.sol

```diff
@@ -242,6 +242,29 @@ interface ILensHub {
      */
     function followWithSig(DataTypes.FollowWithSigData calldata vars) external;
 
+    /**
+     * @notice Toggle Follows on the given profiles, emiting toggle event for each Follow profile.
+     *
+     * NOTE: `profileIds`, `followNFTIds` and `enables` arrays must be of the same length.
+     *
+     * @param profileIds The token ID array of the profiles to follow.
```

Comment:
These descriptions differ to the ones in `ToggleFollowWithSigData`. Would be nice to make them the same.